opendatahub-io/data-science-pipelines-operator
# Parameterize Requeue Time

## The issue resolved by this Pull Request:
Resolves https://github.com/opendatahub-io/data-science-pipelines-operator/issues/316

## Description of your changes:


Parameterizing the DSPAReconciler requeue time value whilst setting the default to 2m, to allow for more custom requeue times.

## Testing instructions

Set the variable `REQUEUE_TIME` , run DSPO, create a DSPA, and ensure the DSPA deploys correctly.
## Checklist
